docker-entrypoint.sh

```diff
@@ -2,11 +2,18 @@
 set -e
 
 mkdir -p $TEAMCITY_DATA_PATH/lib/jdbc $TEAMCITY_DATA_PATH/config
-if [ ! -f "$TEAMCITY_DATA_PATH/lib/jdbc/postgresql-9.3-1103.jdbc41.jar" ];
+
+if [ "$DATABASE" == "mysql" ];
+then
+    if [ ! -f "$TEAMCITY_DATA_PATH"/lib/jdbc/mysql-connector-java.jar ];
```

Comment:
I hear you and Maven is pretty great. 

The nice thing with using an OS package is that we don't have to hardcode a version. We can rely on the OS maintainers to provide a secure and stable version.